TheWarWithin/DruidFeral.lua

```diff
@@ -1162,10 +1169,15 @@ spec:RegisterHook( "runHandler", function( ability )
         break_stealth()
     end
 
-    if buff.ravenous_frenzy.up and ability ~= "ravenous_frenzy" then
+    if talent.aggravate_wounds.enabled and debuff.dreadful_wound.up and a.spendType == "energy" and a.spend > 0 then
```

Comment:
This should probably be in a `spend` hook but I'll allow it.

Reply:
Wasn't sure if I should make one or not since there already appears to be a combo point specific spender function.

Reply:
They ultimately share the same spend hook, so checking `amt` and ... `resource`? ... to do it in the hook is fine.  But you can leave this as-is.